.github/workflows/build-wheels.yml

```diff
@@ -20,7 +20,11 @@ jobs:
         with:
           platforms: all
       - name: Build wheels
-        uses: joerick/cibuildwheel@v1.11.0
+        uses: pypa/cibuildwheel@v2.0.1
+        with:
+          output-dir: ./wheelhouse
         env:
-          CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-*"
+          CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-* cp310-*"
           CIBW_ARCHS_LINUX: auto aarch64
+          CIBW_PRERELEASE_PYTHONS: True
+          CIBW_BEFORE_ALL_LINUX: yum install -y libffi-devel
```

Comment:
Doesn't this run on Ubuntu?

Reply:
cibuildwheel uses the manylinux docker image to build wheels. If we don’t use additional configuration, the default is manylinux2010, it's based on centos.

About why I add `CIBW_BEFORE_ALL_LINUX` to install `libffi`, please see this issue https://github.com/pypa/cibuildwheel/issues/783

Reply:
manylinux2014 is based on unbutu.

Reply:
Ack!